### DISCUSSION TITLE
Frame (container) feature

### DISCUSSION THREAD

**Discussion by skuridin** [excalidraw/excalidraw#6043] [General]
Figma, Sketch, Tldraw have a feature called "frame". It acts like a group, but one can add an element to it simply by drag-and-dropping it. Does it look interesting to anyone else?

![image](https://user-images.githubusercontent.com/929601/209968136-1c2e091e-c9a5-4e9f-8d2a-1f8b22cba012.png)

https://user-images.githubusercontent.com/929601/209967923-4bc8293e-da7b-465a-b40d-e447529737ce.mov

**Comment by dwelle** [comment]
Planned, but there _was_ no issue.

https://github.com/excalidraw/excalidraw/issues/6044

Wanna help?
